contracts/core/RebalancingSetToken.sol

```diff
@@ -231,16 +231,17 @@ contract RebalancingSetToken is
             "RebalancingSetToken.propose: Invalid or disabled PriceLibrary address"
         );
 
-        // Assert price divisor is non-zero, ensuring a positive slope
+        // Check that time to pivot is within in 6 hours to 3 days
         require(
-            _auctionPriceDivisor > 0,
-            "RebalancingSetToken.propose: Price divisor must be positive"
+            _auctionTimeToPivot > 21600 && _auctionTimeToPivot < 259200,
```

Comment:
Just for consistency sake I'd split these up as well. That should be done on my branch if you wanna copy/paste.